spec/classes/init_spec.rb

```diff
@@ -16,14 +16,13 @@
         when 'RedHat'
           config_dir        = '/etc/monit.d'
           service_hasstatus = true
-          case facts[:operatingsystem]
-          when 'Amazon'
-            monit_version = '5'
-            config_file   = '/etc/monit.conf'
-          else
-            monit_version = '5'
-            config_file   = '/etc/monitrc'
-          end
+          monit_version     = '5'
+          config_file = case facts[:operatingsystem]
```

Comment:
This is still a legacy fact
```suggestion
          config_file = case facts[:os]['name']
```

Reply:
i will remove the legacy facts in another PR, somehow changing them to the new syntax breaks tests (at least some of them do) and I would like to do that on a working CI